tests/acceptance/features/bootstrap/TagContext.php

```diff
@@ -126,6 +126,7 @@ public function theUserHasCreatedFollowingTags(string $user, string $fileOrFolde
 	 * @throws Exception
 	 */
 	public function theUserGetsAllAvailableTags(string $user):void {
+		sleep(1);
```

Comment:
wait is necessary?

Reply:
Sorry, I forgot to explain it. some tests without sleep(1) fail because there is reindexing or something else.
`tag.feature:101` after accepting the share, the user does not see the tags for a short time
`tag.feature:212` after deleting space, tags exist for user a short time

Reply:
without sleep: https://drone.owncloud.com/owncloud/ocis/18331/45/6

Reply:
that is annoying. Do we get a 425 here, so we could wait for that?

Reply:
no no, it is GET method `graph/v1.0/extensions/org.libregraph/tags`. We get 200 with (empty or not empty) response

Reply:
that is sad :-( lets add a comment here in the code explaining it and merge this PR